src/FFMpeg/Filters/Video/ResizeFilter.php

```diff
@@ -98,8 +98,10 @@ public function apply(Video $video, VideoInterface $format)
         if (null !== $dimensions) {
             $dimensions = $this->getComputedDimensions($dimensions, $format->getModulus());
 
-            $commands[] = '-s';
-            $commands[] = $dimensions->getWidth() . 'x' . $dimensions->getHeight();
+            // Using Filter to have ordering
+            $commands[] = '-vf';
+            $commands[] = '[in]scale=' . $dimensions->getWidth() . ':' . $dimensions->getHeight() . ' [out]';
```

Comment:
the doc is just simplify for basic user. but for the concept of `vf` is to control the `in` and `out` and then adding filters step by step.
`-v scale=100:100` means `-v "[in] scale=100:100 [out]"` actually.
if you want to scale two times, you will have a command like this:
`-v "[in] scale=100:100 [p1]; [p1] scale=200:200 [out]"` but you cannot do `-v "scale=100:100; scale=200:200" as you are missing the in and out. the filter processor don't know how to do.